src/app/core/header/header.component.css

```diff
@@ -0,0 +1,3 @@
+h2 {
+  flex: 1 1 auto;
+}
```

Comment:
Since we need only one CSS style, we can add it directly to the styles property of the component decorator.